site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -102,7 +102,7 @@ const UsersPage: FC = () => {
 		authMethodsQuery.isLoading ||
 		groupsByUserIdQuery.isLoading;
 
-	if (showOrganizations && location.pathname !== "/deployment/users") {
+	if (location.pathname === "/users") {
```

Comment:
Could I get more context on this change? Mainly wondering why we went from `!==` to `===`

Reply:
The goal is just to always redirect from /users to /deployment/users. I guess I could've left it the other way, this just feels clearer to me.